album/USSW-0369.toml

```
@@ -0,0 +1,32 @@
+[album]
+album_id = "0614f511-2c92-4739-9884-36c88fe92f7c"
+title = "夏夢ノイジー"
+artist = "亜咲花"
+date = 2022-08-10
+type = "normal"
+catalog = "USSW-0369"
+tags = ["サマータイムレンダ"]
```

```suggestion
tags = ["亜咲花"]
```

only M01 belongs to `サマータイムレンダ`